recording-oracle/src/modules/campaigns/campaigns.service.ts

```diff
@@ -491,23 +497,31 @@ export class CampaignsService {
             endDate,
           );
 
-          const dailyReward = this.calculateDailyReward(campaign);
+          let progressValueTarget: number;
+          let progressValue: number;
+          if (isMarketMakingCampaign(campaign)) {
+            progressValueTarget = campaign.details.dailyVolumeTarget;
+            progressValue = (progress.meta as MarketMakingMeta).total_volume;
+          } else if (isHoldingCampaign(campaign)) {
+            progressValueTarget = campaign.details.dailyBalanceTarget;
+            progressValue = (progress.meta as HoldingMeta).total_balance;
+          } else {
+            progressValueTarget = 1;
```

Comment:
I believe this is done in order to trick typescript into thinking that both `progressValueTarget` and `progressValue` are defined but I guess I'd rather go with throwing an error and write some error log so we could track that and create an alret on Datadog.

Wdyt?

Reply:
It wasn't to trick TS, but like: "by default for any new strategy use full reward pool", but we can stick to exception to enforce adding some values here for each new type 👍🏻